IMLS_scraper/imls_scraper.py

```diff
@@ -0,0 +1,105 @@
+import os
+import sys
+import fnmatch
+from datetime import datetime
+from typing import List, Any
+from zipfile import ZipFile
+from io import BytesIO
+import pandas as pd
+import requests
+from bs4 import BeautifulSoup
+import pandas as pd
+import re
+from pymongo import MongoClient, errors
+from tqdm import tqdm
+import urllib 
+
+_i = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+if _i not in sys.path:
+    # add parent directory to sys.path so utils module is accessible
+    sys.path.insert(0, _i)
+del _i  # clean up global name space
+from shared_code.utils import (
+    check_similarity, locate_potential_duplicate,
+    insert_services, client
+)
+from shared_code.base_scraper import BaseScraper
+
+     
+class IMLS_Scraper(BaseScraper):
```

Comment:
Standard for Python to have CamelCase for class names like how BaseScraper is named. It will be great if we all follow same standards.

Reply:
Will do